### DISCUSSION TITLE
Git said You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.  If you meant to apply as a student, please cancel your application and reapply here.

### DISCUSSION THREAD

**Discussion by RonSlow60** [community/community#57495] [GitHub Education]
### Hi everyone,

I applied as a student with my student email and photo of my enrollment confirmation form from TAFE NEW as evidence/
Not good enough.
I got this response back by email...

===
Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Jun 7, 2023
You are unlikely to be verified until you have completed your [GitHub user profile] with your full name as it appears in your academic affiliation document plus a short bio.
If you meant to apply as a student, please cancel your application and reapply [here]
===

I DID apply as a student (never anything else).
So, now I've filled in my Bio (not that the application said you have to)
So ... how do I reapply?  I used the links in the email.  No good.  It just has the same form I filled in but with a message on the right saying
===
Unfortunately, we weren't able to approve your educational discount request for...
What happened?
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:
You are unlikely to be verified until you have completed your [GitHub user profile] with your full name as it appears in your academic affiliation document plus a short bio.
If you meant to apply as a student, please cancel your application and reapply [here]
===
Anyway, I get nowhere, because nothing happens when I click on [ CONTINUE ] button.

How do I reapply now I've filled in the bio (that they didn't say to fill in in the first place)?

### A suggestion...(optional)

Let me apply

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
